example/lib/assurance.dart

```diff
@@ -0,0 +1,96 @@
+import 'dart:async';
+import 'dart:developer';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_aepcore/flutter_aepcore.dart';
+import 'package:flutter_aepassurance/flutter_aepassurance.dart';
+
+class assurancePage extends StatefulWidget {
+  @override
+  _MyAppState createState() => _MyAppState();
+}
+
+class _MyAppState extends State<assurancePage> {
+  String _assuranceVersion = 'Unknown';
+  String _urlText = '';
+
+  @override
+  void initState() {
+    super.initState();
+    initPlatformState();
+  }
+
+  // Platform messages are asynchronous, so we initialize in an async method.
+  Future<void> initPlatformState() async {
+    late String assuranceVersion;
+
+    // Platform messages may fail, so we use a try/catch PlatformException.
+    try {
+      assuranceVersion = await Assurance.extensionVersion;
+    } on PlatformException {
+      log("Failed to get Assurance extension versions");
+    }
+
+    // If the widget was removed from the tree while the asynchronous platform
+    // message was in flight, we want to discard the reply rather than calling
+    // setState to update our non-existent appearance.
+    if (!mounted) return;
+
+    setState(() {
+      _assuranceVersion = assuranceVersion;
+    });
+  }
+
+  // UTIL
+  RichText getRichText(String label, String value) {
+    return new RichText(
+      text: new TextSpan(
+        // Note: Styles for TextSpans must be explicitly defined.
+        // Child text spans will inherit styles from parent
+        style: new TextStyle(
+          fontSize: 14.0,
+          color: Colors.black,
+        ),
+        children: <TextSpan>[
+          new TextSpan(
+              text: label, style: new TextStyle(fontWeight: FontWeight.bold)),
+          new TextSpan(text: value),
+        ],
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) => Scaffold(
+        appBar: AppBar(title: Text("Assurance Screen")),
+        body: Center(
+          child: ListView(shrinkWrap: true, children: <Widget>[
+            Text('AEPAssurance version = $_assuranceVersion\n'),
+            ElevatedButton(
+              child: Text("MobileCore.trackState(...)"),
```

Comment:
is this Analytics page or Assurance page? I think assurance configuration can reside on the main page since it is something used by all the extensions.

Reply:
I'd say we should move these track examples to MobileCore and keep Assurance for the connect only, you can also cleanup the imports after that.

Reply:
I remove the Analytics track calls from the Assurance's screen.  I think we had that there for ACPAssurance while it can't access to other extensions. For mono repo, once setup the Assurance link, all the API in the other screens can be also tracked.
I don't think we need to put the MobileCore.trackState and trackAction to the MobileCore Screen since we are not supporting Analytics in Flutter right? @cdhoffmann

Reply:
I agree we should probably just remove the trackState and trackAction until we have Analytics support @cacheung.